### PR TITLE
Update for Arduino MKR series

### PR DESCRIPTION
**Pull request by Mauroalfieri**
Hi,
I updated your library to expand the number of boards to the arduino MKR family.

Mauro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,40 +20,27 @@ EPD | Controller
...
-Busy | 27
-
 ## Program
 See [examples](./examples).
 
 # License
 MIT
 
 # References
-- [ESP32で1.54インチのePaperを動かす方法](http://asukiaaa.blogspot.com/2019/01/esp32154epaper.html)
```
Comment (asukiaaa) [inline README.md]: Why did you remove this link?
For ESP32-WROOM-32 `GPIO6 to GPIO11 are connectedto the integrated SPI flash integrated on the module and are not recommended for other uses.`
https://www.espressif.com/sites/default/files/documentation/esp32-wroom-32_datasheet_en.pdf

**Comment by asukiaaa** [comment]
Thank you for creating a pull request.
I add a comment about readme.
I approve modifications except for readme.

File: README.md (hunk 1 above)
Comment (Mauroalfieri) [reply README.md]: I'm sorry, I lost the link unintentionally

**Comment by Mauroalfieri** [comment]
Hi, I update my branch with the original README file.

**Comment by asukiaaa** [comment]
Thank you for updating.
